INSTALL.md

```diff
@@ -55,10 +53,10 @@ To compile from source, open a **Terminal (Linux/MacOS)**, the **MSVC Tools Comm
    ```
```

Comment:
I just noticed that the point above recommends using the `latest` git tag, which however currently points to version 2.7.4...

Reply:
You may need to pull tags on your computer, latest points to 2.7.9 on the repository

Reply:
Oops, indeed, sorry.